src/renderLatex.ts

```diff
@@ -16,7 +16,8 @@ export default function renderLatexInTextAsHTMLString(text: string, delimiters:
       const latex = data[i].data;
       const displayMode = data[i].display;
       try {
-        const rendered = renderToString(latex, { displayMode });
+        const options = macros ? { displayMode, macros } : { displayMode }
```

Comment:
can't you just pass `{ displayMode, macros }`, does `katex` throw if `macros: undefined` is passed?

Reply:
The code `const rendered = renderToString(latex, { displayMode, macros });` gives an error, 
>semantic error TS2345 Argument of type '{ displayMode: boolean | undefined; macros: object | undefined; }' is not assignable to parameter of type 'KatexOptions | undefined'.
  Object literal may only specify known properties, and 'macros' does not exist in type 'KatexOptions | undefined'.

I am a typescript newbie, how do I resolve it?